### DISCUSSION TITLE
failed escalating privilege from 'exec' to 'privilege_exec'

### DISCUSSION THREAD

**Discussion by zysm0** [scrapli/nornir_scrapli#124] [General]
Hi guys

I'm performing a course about Nornir and when i'm executing this script:

from nornir import InitNornir
from nornir_scrapli.tasks import send_command
from nornir_utils.plugins.functions import print_result

nr = InitNornir(config_file="config.yaml")

results = nr.run(task=send_command, command="show version")
print_result(results)

I get this error:

scrapli.exceptions.ScrapliAuthenticationFailed: failed escalating privilege from 'exec' to 'privilege_exec'. do you need to set an 'auth_secondary' password?

Really i don't understand where is the issue because i'm starting in this world of programming.

Thanks a lot!

**Comment by carlmontanari** [comment]
you need to set `auth_secondary` under `extras` in your inventory. I'm assuming if/when you ssh to your device you end up at an exec (>) prompt. Scrapli needs to know the enable password.
